Examples/Movies/getStyleFromScore.js

```diff
@@ -13,19 +13,17 @@
  *
  * @flow
  */
-'use strict';
 
-var React = require('react-native');
-var {
+import React, {
   StyleSheet,
-} = React;
+} from 'react-native';
 
-var MAX_VALUE = 200;
+const MAX_VALUE = 200;
 
 import type { StyleObj } from 'StyleSheetTypes';
 
-function getStyleFromScore(score: number): StyleObj {
-  if (score < 0) {
+export default function getStyleFromScore(score: number): StyleObj {
+  if (!score || score < 0) {
```

Comment:
Is this check necessary? Is a null/false score ever passed in? The flow definition claims it's always defined.